README.md

```diff
@@ -77,17 +77,17 @@ This will automatically create boundaries and define your public interface for e
 
 
 ### Advanced
-Modguard also supports specific allow lists within `public`.
+Modguard also supports specific allow lists within `public`. The `allowlist` parameter accepts a list of strings and regex expressions.
 ```python
-@modguard.public(allowlist=['utils.helpers'])
+@modguard.public(allowlist=['utils.helpers', r'.*aux.*'])
```

Comment:
Might be more readable to use a realistic example like `r'core\.project\.*'` or similar